### PR TITLE
[Feature #159521401] Finance Involvement in Redemption Requests

### PR DESCRIPTION
**Pull request by Andretalik**
## Description (what problem you're fixing)

 - The finance dept. needed the ability to change the status of a redemption request to completed once the funds have already been sent out.
 - They also needed to be notified when a redemption request is approved by the CIO so they can begin processing payment.

## Fix (what you did to fix it)

  - Added notifications that will be emailed to the relevant center for the Redemption Request.
  - Added functionality that only Finance members can change the status of a redemption request to completed.

## How to test (describe how to test your PR)

- Clone the repository, install docker, run `make test` within the repository.
- Trigger a build on CircleCI.

### CODE REVIEW

**Comment by kimani-ndegwa** [comment]
Do rebase this @Andretalik

File: src/api/endpoints/redemption_requests.py [hunk 1]
```diff
@@ -237,13 +237,21 @@ def put(cls, redeem_id=None):
             society.used_points = redemp_request
             redemp_request.status = status
 
+            # Get the relevant Finance Center to respond on RedemptionRequest
+            if str(redemp_request.center.name.lower()) == 'kampala':
+                finance_email = str(redemp_request.center.name.lower())+".fin"
+                "ance@andela.com"
```
[the remark this answers is not in the document]
Comment (kimani-ndegwa) [reply src/api/endpoints/redemption_requests.py]: Is this a pep8 rule that is causing this oddity or what is up with this? @Andretalik @rnjane @georgreen @zachang @solnsumei @LarryWachira

[the remark this answers is not in the document]
Comment (georgreen) [reply src/api/endpoints/redemption_requests.py]: @Andretalik it seems you are trying to convert a string into a string :sweat_smile:
```python
     str(redemp_request.center.name.lower()) == 'kampala'
```
redemp_request.center.name  is a string already

[the remark this answers is not in the document]
Comment (LarryWachira) [reply src/api/endpoints/redemption_requests.py]: +1 @georgreen. `.lower()` wouldn't raise hell if `redemp_request.center.name` wasn't a string. We'll need to remember to remove `str()`.

**Pull request merged**